devel/meson/Portfile

```diff
@@ -4,7 +4,9 @@ PortSystem          1.0
 PortGroup           python 1.0
 PortGroup           github 1.0
 
-github.setup        mesonbuild meson 0.52.0
+github.setup        mesonbuild meson 0.52.1
+revision            0
```

Comment:
patch revision should be 1 as you apply additional patch

Reply:
normally that would be true, but because we're bumping the version to 0.52.1, that will force the rebuild so bumping the revision is not needed. 

Indeed I had bumped the revision to 1 before, but then we decided to increase the version so changed it back to 0. This is why this line is here at all -- we should just leave this line in now, as it belongs here, but set it zero.